lib/search_files.js

```diff
@@ -0,0 +1,291 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var glob = require('glob');
+
+
+function isWindows() {
+  return process.platform === 'win32';
+}
+
+function isFunction(v) {
+  return (typeof v === 'function');
+}
+
+function isObject(v) {
```

Comment:
On my PC, load time of lodash was about 50ms, and total load time of lodash.isobject, .isarray, .isstring, .isfunction, .identity, and .mapvalues was about 10ms.

Should we use only needed function's modules of lodash?